.vib/tensorflow-serving/goss/vars.yaml

```diff
@@ -0,0 +1,15 @@
+binaries:
+  - render-template
+directories:
+  - mode: "0775"
```

Comment:
You should also check the `user` and `group` ownership, as specified by:

https://github.com/bitnami/containers/blob/813d973dbed1aba659544d5cc9ccbf43ae900b03/bitnami/tensorflow-serving/2/debian-11/rootfs/opt/bitnami/scripts/tensorflow-serving/postunpack.sh#L22